.github/workflows/continuous-integration.yaml

```diff
@@ -36,11 +36,11 @@ jobs:
           - php: '7.2'
             elasticsearch: '7.2.1'
           - php: '7.3'
-            elasticsearch: '7.3.2'
+            elasticsearch: '7.10.2'
```

Comment:
@deguif Can you remind me how the older ES versions are related to older PHP versions?

Reply:
What do you mean about `related to older PHP versions`?

Reply:
Why do we test PHP 7.2 with ES 7.1.1 and ES 7.2.1 but PHP 7.3 only with ES 7.3.2?

Reply:
IIRC php 7.2 is the lowest PHP version supported by this library.
Maybe we could test it against:
- the lowest PHP version we support
- the latest version available of PHP 7.4 (and 8.0, if we support PHP 8.x)

WDYT?

Reply:
@ruflin I think it was done to test several ES versions but I'm not sure, but each ES version is not tied to a PHP one, just the matrix was done this way.

We already test it with PHP 8.0 but elasticsearch official library has not yet published a stable version supporting PHP 8.0.

Reply:
I definitively like the idea of testing multiple PHP versions and I think we should keep that. The part I'm challenging is aorund how many ES versions we should test in parallel. My suggestion would be to test against the most recent version released by Elastic against the available / supported PHP version. We might add one more test against the most recent ES version which we things is fully compatible and does not require special testing code, so that would be currently 7.4 I think but might soon be 7.10 so not sure it is worth it.

The alternative is to test the first and last minor of a major, like 7.0.x and 7.10.2 today to make sure we stay compatible, but I think this creates overhead on the testing side and we should rely on Elastic to keep things mostly backward compatible during minor releases.

Reply:
I suggest to do this in a follow up PR.